packages/react-cli/src/commands/fetch-configs.ts

```diff
@@ -0,0 +1,29 @@
+import { Command } from '@oclif/command';
+import { ensureDependencies } from '../tasks/ensure-dependiencies';
+import Listr from 'listr';
+import { downloadTSConfigFiles } from '../tasks/download-ts-config';
+import { exec } from '../utils/shell';
+
+export class FetchConfigsCommand extends Command {
+  static description = 'Fetches config files for linting and typescript projects';
+
+  static examples = ['$ react fetch-configs'];
+
+  async run() {
+    await ensureDependencies();
+
+    let path = './';
+    let tasks = new Listr([
```

Comment:
I would prefer `const` here and when defining the path variable above since these are never reassigned

Reply:
we don't have C.I. / linting setup :-P